tests/common/test_common.py

```diff
@@ -52,9 +52,15 @@ def test_check_year():
     assert check_year("file_140032021.csv") == "2021"
     assert check_year("file_2017-18.csv") == "2018"
     assert check_year("file_201819.csv") == "2019"
+    assert check_year("file_1920.csv") == "2020"
```

Comment:
why would "1920" imply the year 2020 instead of the year 1920.

I understand how your code does the conversion...just not why you chose to interpret it that way.

My first guess would have been that you are assuming the format included the month number as the first two characters and the year number as the last two characters such that 0822 == August 2022. 
However, there is no 19th month so I'm not sure how 1920 will be read. If "20" is the year, what does 19 represent?. @patrick-troy

Reply:
1920 implied 2019-2020 and we want the latest year which would be 2020 in this case, much like 2122 would become 2022. We can only accept data that is no more than 6 years old so we should not get any years starting 19xx. This will also break from the year 21xx onwards but my hope is there is another system in place by then

Reply:
Ohhh 
I see.